usr/share/lib/img_proof/tests/SLES/test_sles_hardened.py

```diff
@@ -23,6 +24,9 @@ def setup_swap(host, swap_file=SWAP_FILE):
         ]:
             result = host.run(command)
             if result.rc != 0:
+                print("{} command failed".format(command), file=sys.stderr)
```

Comment:
Please drop the file descriptor override `file=sys.stderr`. This prevents img-proof from catching the output and putting it in the log file.

Reply:
From command line this is visible by running `img-proof results show -l`. And it also can be retrieved when running from code.